### DISCUSSION TITLE
Question about text under "Personal History"

### DISCUSSION THREAD

**Discussion by malcolmbastien** [binarybottle/engram-layout#7] [General]
Hi Arno, following the layout's development closely and would like to say great job with everything you've accomplished so far.

I had a question about this passage in the ReadMe under "Personal History":
> While these technologies can significantly improve comfort and reduce strain, an optimized key layout can only help when typing on ergonomic or standard keyboards.

Before this sentence, you mentioned voice dictation, footswitches, etc. But this last part confuses me. This is trying to say that an optimized keyboard layout only works when you use a keyboard?

**Comment by binarybottle** [comment]
Thank you for your interest and for pointing out this most unclear sentence!  I have replaced it with the following;

> While these technologies can significantly improve comfort and reduce strain, 
if you have to type on a keyboard, it can only help to use a key layout optimized according to sound ergonomics principles.

**Comment by binarybottle** [comment]
Please see https://github.com/binarybottle/engram-2021/discussions/86#discussion-10126586 for an announcement of the new Engram approach/layouts/repo!
